app/frontend/src/components/Inputs/Calendar.tsx

```diff
@@ -25,33 +29,27 @@ interface CalendarRowProps extends React.PropsWithChildren {
 }
 
 interface CalendarCellProps {
-  columnNum: number;
-  onChange: (selected: boolean) => any;
-  rowNum: number;
-  selected?: boolean;
+  key: number;
+  data: string;
+  cell: any;
```

Comment:
Is it possible to limit the typing for `cell`? I see that below is is set as

```js
{
  row: number
  column: number
}
```

Reply:
Done.